docs/index.md

```diff
@@ -0,0 +1,38 @@
+# OPTIMADE Python Tools
```

Comment:
Could we make this a symlink to the top-level `./README.md`, or has something changed?

Reply:
Didn't even realize github worked with symlinks.